xml/System.Data/DataReaderExtensions.xml

```diff
@@ -275,11 +288,13 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <param name="name">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="reader">The <see cref="T:System.Data.Common.DbDataReader" /> to get the column value from.</param>
+        <param name="name">The name of the column.</param>
+        <summary>Gets the value of the specified column as a <see cref="T:System.Decimal" /> object.</summary>
+        <returns>The value of the specified column.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.InvalidCastException">The specified cast is not valid.</exception>
+        <related type="Article" href="~/docs/framework/data/adonet/ado-net-overview.md">ADO.NET Overview</related>
```

Comment:
This article isn't super relevant but it is in most methods in DbDataReader. Remove?